polling_stations/apps/data_finder/directions_clients.py

```diff
@@ -0,0 +1,129 @@
+import abc
+import json
+import requests
+import urllib
+from collections import namedtuple
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+
+Directions = namedtuple('Directions', [
+    'walk_time', 'walk_dist', 'route', 'precision', 'source'])
+
+
+class DirectionsException(Exception):
+    pass
+
+
+class DirectionsClient(metaclass=abc.ABCMeta):
+
+    def __init__(self):
+        pass
```

Comment:
Still don't have my head around `abc`, but is this correct? Should it call `super`?